ocs_ci/ocs/registry.py

```diff
@@ -68,10 +68,10 @@ def get_registry_pod_obj():
             break
         elif len(pod_obj) == 0:
             raise UnexpectedBehaviour("Image-registry pod not present")
-        elif iteration > 5:
+        elif iteration == 18:  # 18*10 = 180 secs.
             raise UnexpectedBehaviour("Waited for 3 mins Image-registry pod is not in Running state")
         else:
-            logger.info(f"Waiting for 30 sec's for registry pod to be up iteration {iteration}")
+            logger.info(f"Waiting for {wait_time} seconds for registry pod to be up - iteration {iteration}")
```

Comment:
This line is pretty long as I can see it's breaking in other line in github UI view. Can you please split like:
```python
logger.info(
    f"Waiting for {wait_time} seconds for registry pod to be up - iteration {iteration}
")
```

Reply:
<rant>
Really hate it when I need to break lines because of *Github* UI
</rant>

Reply:
It's not just cause of github UI but also cause of our conventions described here:
https://ocs-ci.readthedocs.io/en/latest/docs/coding_guidelines.html
*Line length should be maximum 79 characters!*

I know it's not 100% followed and we do not prohibit it pep8 check in the tox as some people just not believe in 79 chars. So I am trying to at least keep eyes when I see it's too long and I see it overflows in UI.

Reply:
flake8 did not complain on it. I don't mind changing it of course. I really just wanted to change the timeout...

Reply:
I triggered verification job with tier1 in our jenkins so it will update status here how it passed to be sure we are OK.

Yes, please change. I also sent you invitation to red-hat-storage organization on github, you will need to accept from email most likely. But that's probably not the case of complaining DCO.

Did you have such issue also in different projects or just this one PR?

Maybe if you will do the change in the commit and commit --amend your changes and for sure once more:

git commit --amend --signoff
Now your commits will have your sign off. Next run

git push --force-with-lease origin mykaul-registry_iterate
Commit sha: fad6a04, Author: Yaniv Kaul, Committer: Yaniv Kaul; The sign-off is missing.

Maybe DCO will be satisfied.